project-iam/roles/ec2_instance_profile.tf

```diff
@@ -35,7 +35,10 @@ resource "aws_iam_policy" "policy" {
             "Effect": "Allow",
             "Action": [
               "ecs:*",
-              "ecr:*"
+              "ecr:*",
+              "kms:*",
+              "s3:GetObject",
+              "s3:GetBucketLocation"
```

Comment:
Please distinguish those two priviliges to separate Statement (you can add it in curly brackets after adding a comma) so it could access only desired bucket.
```
{
            "Effect": "Allow",
            "Action": [
                "s3:GetObject",
                "s3:GetBucketLocation"
            ],
            "Resource": [
                "arn:aws:s3:::programmers-only-artifacts/*",
                "arn:aws:s3:::programmers-only-artifacts"
            ]
        }
```

Reply:
Added.